.github/workflows/report-bundle-size.yaml

```diff
@@ -0,0 +1,86 @@
+name: Report Bundle Size
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  bundle-size:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout PR branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          persist-credentials: false
+
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: zulu
+          java-version: 21
+
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y python3 jq gzip ffmpeg
+
+      - name: Build bundle
+        run: python3 build/all.py
+
+      - name: Compute bundle size
+        id: size
+        run: |
+          FILE="dist/shaka-player.compiled.js"
+
+          if [ ! -f "$FILE" ]; then
+            echo "Bundle not found: $FILE"
+            exit 1
+          fi
+
+          RAW_KB=$(du -k "$FILE" | cut -f1)
+          GZIP_B=$(gzip -c "$FILE" | wc -c)
+          GZIP_KB=$(echo "$GZIP_B / 1024" | bc)
+
+          echo "raw_kb=$RAW_KB" >> $GITHUB_OUTPUT
+          echo "gzip_kb=$GZIP_KB" >> $GITHUB_OUTPUT
+
+      - name: Add or Update PR Comment
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.SHAKA_BOT_TOKEN }}
```

Comment:
@joeyparrish ?

Reply:
This can't be used from a pull_request trigger.  Secrets can only be accessed from a pull_request_target trigger, and that means the workflow itself is always run from `main` and not from the PR author.  (So the author can't modify the workflow to access secrets without it being reviewed or merged.)